docs/pages/config-reference.mdx

```diff
@@ -302,6 +302,12 @@ auth_service:
     # inactivity. (Currently only supported for SSH connections)
     idle_timeout_message: ""
 
+    # Configures a banner message to be displayed to a user logging into the cluster, 
```

Comment:
Not sure what resource this would go into, I think `idle_timeout_message` should go into `ClusterNetworkingConfig` to be paired with an `idle_timeout`, but this one is a bit of a odd one.

Reply:
Spoke with @klizhentas - upshot of conversation was that it's probably overkill to break it out into its own resource and this is the least-bad place for it.